### PR TITLE
shed: util: get all msig

### PR DESCRIPTION
**Pull request by jennijuju**
image

## Checklist

Before you mark the PR ready for review, please make sure that:
- [ ] All commits have a clear commit message.
- [ ] The PR title is in the form of of `<PR type>: <area>: <change being made>`
    - example: ` fix: mempool: Introduce a cache for valid signatures`
    - `PR type`: _fix_, _feat_, _INTERFACE BREAKING CHANGE_, _CONSENSUS BREAKING_, _build_, _chore_, _ci_, _docs_,_perf_, _refactor_, _revert_, _style_, _test_
    - `area`: _api_, _chain_, _state_, _vm_, _data transfer_, _market_, _mempool_, _message_, _block production_, _multisig_, _networking_, _paychan_, _proving_, _sealing_, _wallet_, _deps_
- [ ] This PR has tests for new functionality or change in behaviour
- [ ] If new user-facing features are introduced, clear usage guidelines and / or documentation updates should be included in https://lotus.filecoin.io or [Discussion Tutorials.](https://github.com/filecoin-project/lotus/discussions/categories/tutorials)
- [ ] CI is green

### CODE REVIEW

File: cmd/lotus-shed/msig.go [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+	Action: func(cctx *cli.Context) error {
+		api, closer, err := GetFullNodeAPI(cctx)
+		if err != nil {
+			return err
+		}
+		defer closer()
+
+		ctx := ReqContext(cctx)
+
+		ts, err := LoadTipSet(ctx, cctx, api)
```
Comment (magik6k) [inline cmd/lotus-shed/msig.go:121]: We can just call the one in `cli/` package like we do in other shed commands

File: cmd/lotus-shed/msig.go [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+		},
+	},
+	Subcommands: []*cli.Command{
+		multisigGetAllCmd,
+	},
+}
+
+var multisigGetAllCmd = &cli.Command{
+	Name:  "all",
+	Usage: "get all multisig actor on chain with id, siigners, threshold and balance",
```
Comment (magik6k) [inline cmd/lotus-shed/msig.go:105]:
```suggestion
	Usage: "get all multisig actor on chain with id, signers, threshold and balance",
```

**Review by arajasek — CHANGES_REQUESTED** [review]
This can be _significantly_ faster (order of seconds) if we change it to work in "offline" mode -- instead of querying a daemon's API, it can work on the daemon's blockstore directly, iterating over all the actors in the state tree. See [here](https://github.com/filecoin-project/lotus/blob/e65fae28de2a44947dd24af8c7dafcade29af1a4/cmd/lotus-shed/miner-types.go#L107) for an example shed command.

**Review by arajasek — APPROVED** [review]
Nits but LGTM

**Comment by arajasek** [comment]
And needs `make jen`

**Pull request merged**